### ISSUE TITLE
Import/export redirects

### ISSUE THREAD

**Issue by Cappiebara** [TheNamlessGuy/browser-redirector#6]
Hello! I love this extension, it works the best of those I try, but other redirect extension has a really useful feature where you can export to a JSON file and import it. So you can share with friends or other browser without copy/paste a hundred things. Wrote some code for this myself, but it would be better if it were built into the options page.

```js
// EXPORT

const listener = (response) => {
  const s = JSON.stringify(response);
  console.log(s);
}

const port = browser.runtime.connect();

port.onMessage.addListener(listener);
port.postMessage({ action: "get-options", ...{}});
```

```js
// IMPORT

const YOUR_JSON = "{}"; // Change this to match your JSON

const listener = (response) => {
	const s = JSON.stringify(response);
	console.log(s);
}

const port = browser.runtime.connect();
const redirects = JSON.parse(YOUR_JSON);

port.onMessage.addListener(listener);
port.postMessage({ action: "set-options", ...redirects});
```

**Comment by TheNamlessGuy** [comment]
Sure, makes sense.

**Comment by TheNamlessGuy** [comment]
Should be fixed in 0.7.0
